Assets/Scripts/PlayerController/PlayerMovement.cs

```diff
@@ -37,16 +40,36 @@ public class PlayerMovement : MonoBehaviour
     private float m_LandingTime = 0;
     private bool m_IsMidAir;
     private bool m_JumpedInCurrentFrame;
+    public bool m_cannotMove; 
+    private float m_bombDelay;
 
     void Start()
     {
         AetherInput.GetPlayerActions().Jump.performed += HandleJump;
         m_CharacterController = GetComponent<CharacterController>();
+        m_cannotMove = false;
     }
 
     // Update is called once per frame
     void Update()
     {
+
+        if (m_bombDelay > 0.0f) 
+        {
+            m_bombDelay -= Time.deltaTime;
+        }
+        // Temporary Hacky Fix 
+        // TODO: Exploded while in air 
+        if (m_cannotMove) 
+        {
+            if (!GetIsGrounded()) 
+            {
+                 m_CharacterController.Move(new Vector3(-1.0f, -1.0f, -1.0f));
+            } 
+
+            return;
+        }
+
```

Comment:
You might want to isolate bomb logic into a separate class. Let `PlayerMovement` handle just the core movement of the player.

Reply:
Will do after fixing the other branch on powerups!